wsd/Storage.cpp

```diff
@@ -274,10 +274,26 @@ std::unique_ptr<StorageBase> StorageBase::create(const Poco::URI& uri, const std
     {
         LOG_INF("Public URI [" << LOOLWSD::anonymizeUrl(uri.toString()) << "] considered WOPI.");
         const auto& targetHost = uri.getHost();
+        bool allowed(false);
         if (WopiHosts.match(targetHost) || isLocalhost(targetHost))
         {
-            return std::unique_ptr<StorageBase>(new WopiStorage(uri, jailRoot, jailPath));
+            allowed = true;
+        }
```

Comment:
This could simplify to:

`bool allowed = WopiHosts.match(targetHost) || isLocalhost(targetHost);`